integration_tests/test_tar_plugin.py

```diff
@@ -40,7 +40,8 @@ def test_stage_nil_plugin(self):
             'notop',
             'parent',
             'slash',
-            'readonly_file'
+            'readonly_file',
+            os.path.join('destdir1','destdir2','top-simple')
```

Comment:
You are missing spaces between the commas here. I wonder why the static tests are not catching this pep8 error.
os.path.join('destdir1', 'destdir2', 'top-simple')